src/pretix/control/templates/pretixcontrol/event/mail.html

```diff
@@ -92,7 +114,7 @@ <h4 class="panel-title">
                             </a>
                         </h4>
                     </div>
-                    <div id="order_expirew" class="panel-collapse collapse">
+                    <div id="order_expire" class="panel-collapse collapse">
```

Comment:
oops! I thought it's just a typo 😭 
Thanks for your reminder, I forgot to change the ref. as well. I'll just change it back!